task_template/entrypoint.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 echo "$SSL_CERTIFICATE" >> cert.pem
 echo "$SSL_KEY" >> key.pem
-uvicorn main:app --ssl-keyfile /usr/project/key.pem --ssl-certfile /usr/project/cert.pem --port 8062 --host 0.0.0.0
\ No newline at end of file
+uvicorn main:app --ssl-keyfile /usr/project/key.pem --ssl-certfile /usr/project/cert.pem --port 8062 --host 0.0.0.0
```

Comment:
While we currently do have an issue with the deployer, we have been moving the SSL layer to the nginx ingress on the deployment kubernetes. Thus, the individual apps do no longer need to host the certificates themselves. 
```suggestion
uvicorn main:app --port 8062 --host 0.0.0.0
```